### PR TITLE
added vendorservice methods and set service methods and added to data…

### PR DESCRIPTION
**Pull request by jayharris14**
…base

### CODE REVIEW

File: src/com/main/QuickBytes.java [hunk 1]
```diff
@@ -1,10 +1,12 @@
 package com.main;
 
+import java.sql.SQLException;
+
 import com.main.service.UserService;
 
 public class QuickBytes {
 	
-	public static void main(String[] args) {
+	public static void main(String[] args) throws SQLException {
```
Comment (Kev-Index) [inline src/com/main/QuickBytes.java]: Why do we throw instead of catch?

File: src/com/main/model/Set.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+		this.available = available;
+		this.vendorId = vendorId;
+	}
+	public int getSetId() {
+		return setId;
+	}
+	public void setSetId(int setId) {
+		this.setId = setId;
+	}
+	public Set(String name, float price, int available, int vendorId) {
```
Comment (Kev-Index) [inline src/com/main/model/Set.java:17]: Can you move this under the first constructor to keep all the constructors together?

File: src/com/main/service/ItemService.java [hunk 3]
```diff
@@ -75,5 +75,9 @@ public void editItem(int vendorId) {
 		db.updateItem(i);
 			
 		
+	}
+	public void deleteItem() {
```
Comment (Kev-Index) [inline src/com/main/service/ItemService.java:79]: Don't need a function we're not using

File: src/com/main/service/VendorService.java [hunk 4]
```diff
@@ -37,24 +40,177 @@ public void createAccount() {
 		db.addVendor(vendor);
 	}
 	
-	public void login() {
+	//Jay- allow vendor to login
+	public void login() throws SQLException {
```
Comment (Kev-Index) [inline src/com/main/service/VendorService.java:44]: Why are we throwing exception here?

File: src/com/main/db/Database.java [hunk 5]
```diff
@@ -317,23 +318,147 @@ public void addOrderItem(Order order, Item item) {
...
+		while(rst.next()) {
+			if(username.equals(rst.getString("username"))) {
+				if (password.equals(rst.getString("password"))){
+					result=true;
+				}
+			}
+				
+			}
+			
+		return result;
```
Comment (Jlpowellv) [inline src/com/main/db/Database.java:361]: don't forget dbClose()

**Pull request closed without merging**